### DISCUSSION TITLE
Task/Automation - Monthly Schedule to include day of week

### DISCUSSION THREAD

**Discussion by Royel-Payne** [gameserverapp/Platform#261] [Suggestions]
I would like to be able to schedule a task to run monthly on a specific day of the week.

Examples:
First Friday of each month
Second Monday of each month

**Comment by mvaessen** [comment]
Added to #517

**Comment by mvaessen** [comment]
Hi, thanks for your suggestion. This is now available on the dashboard.

**Comment by mvaessen** [comment]
Hi, this is now available on the dashboard!
